src/query/frontend/stripped_lexer_constants.hpp

```diff
@@ -208,7 +208,10 @@ const trie::Trie kKeywords = {"union",
                               "websocket",
                               "foreach",
                               "labels",
-                              "edge_types"};
+                              "edge_types",
+                              "off",
+                              "in_memory_transactional",
+                              "in_memory_analytical"};
 
```

Comment:
What does this do? Should I forgot to use it in transactional query maybe :smile: ?

Reply:
Okay, and when do we add words here?

Reply:
@as51340 if you added a new keyword, then you have to also modify this container.